planning/standards-status.md

```diff
@@ -0,0 +1,83 @@
+
+## Security and Authentication
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [MIME](https://www.rfc-editor.org/rfc/rfc2045)                     |  &#10004;      |   &#10004;     |                            | MIME is actually over 3 RFCs: [RFC2045](https://www.rfc-editor.org/rfc/rfc2045), [RFC2046](https://www.rfc-editor.org/rfc/rfc2046), and [RFC2047](https://www.rfc-editor.org/rfc/rfc2047). All 3 are supported for both Desktop and Android platforms. |
+| [List Management Headers](https://www.rfc-editor.org/rfc/rfc2369)  |                |                |                            | See [Bug 29041](https://bugzilla.mozilla.org/show_bug.cgi?id=29041)  |
+| OpenPGP                                                            |  &#10004;      |   &#10004;     | [KaiE]() actively involved |         |
+| S/MIME version?                                                    |  &#10004;      |                |                            |         |
+| [oAuth 2.0](https://datatracker.ietf.org/doc/html/rfc6749)         |  &#10004;      |   &#10004;     |                            | See https://oauth.net/2/  |
+| [LDAP](https://datatracker.ietf.org/doc/html/rfc4511)              |  &#10004;      |      N/A       |                            |         |
+
+## Address Book
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [vCard](https://datatracker.ietf.org/doc/html/rfc6350)             |  &#10004;      |                |                            |         |
+| [CardDAV](https://datatracker.ietf.org/doc/html/rfc4511)           |  &#10004;      |      N/A       |                            |         |
+
+## Calendar
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [iCalendar](https://datatracker.ietf.org/doc/html/rfc5545)         |  &#10004;      |                |                            |         |
+| [CalDAV](https://datatracker.ietf.org/doc/html/rfc4791)            |  &#10004;      |      N/A       |                            |         |
+| [CalDAV Scheduling](https://datatracker.ietf.org/doc/html/rfc6638) |                |      N/A       |                            |         |
```

Comment:
Maybe instead of an empty cell we should have some placeholder showing we have not collected that information yet?

Reply:
Yeah definitely. I'm open to this. Yesterday I discovered that [markdown does not support putting a check box in a table](https://stackoverflow.com/questions/54694160/adding-checkbox-in-markdown-table-does-not-work). At least github markdown and I assume gitbook follows this. That is why I went with the `&#10004;` for the checked boxes. Do you have any suggestions?

Reply:
Yeah, not sure because they don't render emoji in color - maybe gitbook would. I guess we can come up with whatever we want, as long as we provide a legend what the different contents of the cells mean.

Reply:
Right now the legend is:
* `&#10004;` if it's been done and is supported for the given platform
* empty if it has not been done yet
* `N/A` if the standard doesn't apply for the given platform

Given the use of `N/A`, I believe the empty cell intuitively indicates that the item just hasn't been done yet. Another option could be to use `TBD` instead of emptiness? @freaktechnik do you prefer that?

Reply:
I wouldn't call it "hasn't been done yet" though, since that might be misunderstood that it's not been implemented yet. Maybe more like "empty when we haven't collected standards information yet". I think a legend will adequately provide everyone with the needed context though.

Reply:
ok good idea. I've pushed the legend :)